src/features/votes/VoteListItem.tsx

```diff
@@ -0,0 +1,37 @@
+import { LiveVoteCard } from "./LiveVoteCard";
+import { PastEpochCard } from "./PastEpochCard";
+import { useGroupClaimStatus } from "./hooks/useGroupClaimStatus";
+import { Proposal } from "./hooks/useGroupedProposals";
+
+export const VoteListItem = ({
+  proposalGroup,
+}: {
+  proposalGroup: Proposal[];
+}) => {
+  const proposalGroupState = useGroupClaimStatus({
+    proposalGroup,
+  });
+
+  const isLiveEpoch = !proposalGroupState.hasEnded;
+
+  if (isLiveEpoch) {
+    return proposalGroup.map((proposal, i) => {
+      const hasVoted = proposalGroupState.votedForProposal[i];
+      return (
+        <LiveVoteCard
+          hasUserVoted={hasVoted}
+          proposal={proposal}
+          hasStarted={proposalGroupState.hasStarted}
+          key={proposal.id}
+        />
+      );
+    });
+  } else {
+    return (
+      <PastEpochCard
+        proposalGroupState={proposalGroupState}
+        proposalGroup={proposalGroup}
+      />
+    );
+  }
```

Comment:
```suggestion
  }

  return (
    <PastEpochCard
      proposalGroupState={proposalGroupState}
      proposalGroup={proposalGroup}
    />
  );
```

Reply:
Can you explain why this is better?

I understand the second else is not technically needed, but I personally find it nicer to remind the reader that this is not always what is returned.

Reply:
Because it requires less code and looks better, but that's a personal preference.